brave-core/src/test/java/com/github/kristofa/brave/LocalTracerTest.java

```diff
@@ -182,59 +178,33 @@ public void startSpan_with_inheritable_nested_local_spans() {
     }
 
     @Test
-    public void startSpan_nested_local_spans_disabled() {
```

Comment:
removed this test as it is impossible to configure this since the nested flag is not exposed